packages/react/src/themes/teams/staticStyles/globalStyles.ts

```diff
@@ -1,9 +1,6 @@
 import { StaticStyleFunction } from '../../types'
 
 const globalStyles: StaticStyleFunction = siteVars => ({
-  html: {
-    fontSize: siteVars.htmlFontSize,
-  },
   body: {
```

Comment:
I vote for moving all these from `body` to `ProviderBox` (and `popupStyles`) to get rid of an another piece of global styles. Opinions?

Reply:
lets wait but think on this move - wait just because of emerged asks for `ProviderBox` reuse existing DOM element on render

Reply:
Ok, let's think about this later.